pkg/cmdrunner/cmd_runner_test.go

```diff
@@ -14,17 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cmd_runner_test
+package cmdrunner
```

Comment:
Is the lack of `_test` desirable? Would we still not want to consider black box testing here? In general, it always seems much better if we can test the public API. And skimming the rest of the changes in the file, it seems the package name could still be `_test`.

Reply:
Personally I think it's always better to test the public API (ie. exported methods). I find this is less brittle and concentrates testing around how 'your' callers will call the API.

Reply:
I don't think that the test is currently using any functions that aren't public. Rather than bikeshed here and block this PR we can open a separate PR to move this test code into a separate package and discuss there.